info.olasagasti.revelation.json

```diff
@@ -0,0 +1,61 @@
+{
+    "app-id": "info.olasagasti.revelation",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.36",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "revelation",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=xdg-run/gvfs",
+        "--filesystem=home",
+        "--talk-name=org.gtk.vfs",
+        "--talk-name=org.gtk.vfs.*",
+        "--talk-name=org.gnome.ScreenSaver"
+    ],
+    "modules" : [
+        {
+            "name" : "libpwquality",
```

Comment:
Would be nice to add `"cleanup": ["*.a"]` but the file is small so it can be done later.